Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -441,8 +495,9 @@ - (BOOL)calendar:(FSCalendar *)calendar shouldSelectDate:(NSDate *)date atMonthP
         return NO;
     }
 
-    BOOL shouldClearDates = self.selectionConfiguration.allowsMultipleSelection &&
-                            [self.selectionConfiguration shouldClearSelectedDates:self.sortedSelectedDates whenSelectingDate:date];
+    BOOL shouldClearDates = self.isSelectingWholeMonth || (self.selectionConfiguration.allowsMultipleSelection &&
```

Comment:
Should we reset isSelectingWholeMonth in the if (shouldClearDates) statement?

Reply:
Right now `isSelectingWholeMonth` is set to `false` in the methods:
```objc
- (void)calendar:(FSCalendar *)calendar didSelectDate:(NSDate *)date atMonthPosition:(FSCalendarMonthPosition)monthPosition;
- (void)calendar:(FSCalendar *)calendar didDeselectDate:(NSDate *)date atMonthPosition:(FSCalendarMonthPosition)monthPosition;
```
But we could try to move it to this place if you think it is more suitable.

Reply:
No, I think these two places make sense, as that is when a user interacts with the calendar.
Thanks!